test/browser/docs/netlify-cms.js

```diff
@@ -69,7 +69,7 @@ describe( 'Netlify CMS', () => {
       expect( previewPane ).toHaveTextContaining( 'component pages are the best' );
     } );
 
-    it( 'should support switching between the various "show details" tabs', () => {
+    it( 'should support switching between the various "show details" tabs', function() {
```

Comment:
Why is it necessary to strip out the arrow functions? Was there a problem with scope?

Reply:
Nevermind, I see the mocha note.